src/diffusers/schedulers/scheduling_lms_discrete.py

```diff
@@ -348,7 +348,7 @@ def _sigma_to_t(self, sigma, log_sigmas):
         t = t.reshape(sigma.shape)
         return t
 
-    # copied from diffusers.schedulers.scheduling_euler_discrete._convert_to_karras
+    # copied from diffusers.schedulers.scheduling_euler_discrete.EulerDiscreteScheduler._convert_to_karras
```

Comment:
```suggestion
    # Copied from diffusers.schedulers.scheduling_euler_discrete.EulerDiscreteScheduler._convert_to_karras
```

Reply:
I explained this in the PR's first comment. Could you see there?